src/cli/commands.rs

```diff
@@ -0,0 +1,56 @@
+use sc_cli::CliConfiguration;
+use sc_service::Configuration;
+use std::fmt::Debug;
+use std::io::Write;
+use std::{fs, io};
+use structopt::StructOpt;
+
+pub use pallet_abci;
+
+/// The `purge-abci-storage` command used to remove abci storage.
+#[derive(Debug, StructOpt)]
+pub struct PurgeAbciStorageCmd {
+    #[allow(missing_docs)]
+    #[structopt(flatten)]
+    pub shared_params: sc_cli::SharedParams,
+}
+
+impl PurgeAbciStorageCmd {
+    /// Run the purge command.
+    pub fn run(&self, config: &Configuration) -> sc_cli::Result<()> {
```

Comment:
This function should processed the ```-y``` flag during the invoking it. Look at the PureChainCmd srtuct.